disperser/batcher/batcher.go

```diff
@@ -328,6 +328,9 @@ func (b *Batcher) updateConfirmationInfo(
 		requestTime := time.Unix(0, int64(metadata.RequestMetadata.RequestedAt))
 		b.Metrics.ObserveLatency("E2E", float64(time.Since(requestTime).Milliseconds()))
 		b.Metrics.ObserveBlobAge("confirmed", float64(time.Since(requestTime).Milliseconds()))
+		for _, quorumInfo := range batchData.blobHeaders[blobIndex].QuorumInfos {
+			b.Metrics.IncrementBlobSize("confirmed", quorumInfo.QuorumID, int(metadata.RequestMetadata.BlobSize))
```

Comment:
wouldn't the blobSize be identical for eth and eigen?

Reply:
Yes, they will be always the same, since they are by default required. Only custom quorums will have a difference.